src/lib/components/search/load/images/LoadImages.svelte

```diff
@@ -2,22 +2,15 @@
 	// components
 	import LoadImage from '$lib/components/search/load/images/LoadImage.svelte';
 
-	// types
-	import { randomWidth } from '$lib/types/width';
-
 	// variables
 	const numberOfResults: number = 32;
 </script>
 
 <div class="px-4 py-8">
-	<section id="images" class="flex flex-wrap justify-center gap-2">
+	<section id="images" class="grid grid-cols-fit auto-rows-[200px] grid-flow-dense gap-2">
 		{#each { length: numberOfResults } as _, i}
 			<div class="flex-none">
-				{#if i % 3 === 0 && i % Math.floor(2 * Math.random()) !== 0}
-					<LoadImage colored={true} width={randomWidth()} />
-				{:else}
-					<LoadImage width={randomWidth()} />
-				{/if}
+				<LoadImage colored={Math.random() <= 0.1} />
```

Comment:
This doesn't spread the colored images anymore, why the change?

Reply:
Because this kind of randomness aesthetically worked only with the previous layout solution.
I backtracked and remembered your suggestion regarding this. The simpler, the better, as you've said.